setup.py

```diff
@@ -133,6 +133,7 @@ def finalize_options(self):
         'psutil',
         'nibabel',
         'typing',
+        'hypertools'
```

Comment:
Example requirements are not install requirements either, so this is also not necessary.